presto-main/src/main/java/com/facebook/presto/operator/scalar/sql/MapSqlFunctions.java

```diff
@@ -79,7 +80,13 @@ public static String mapTopNValuesComparator()
         return "RETURN IF(n < 0, fail('n must be greater than or equal to 0'), slice(reverse(array_sort(remove_nulls(map_values(input)), f)) || filter(map_values(input), x -> x is null), 1, n))";
     }
 
-    @SqlInvokedScalarFunction(value = "map_remove_null_values", deterministic = true, calledOnNullInput = true)
+    @SqlInvokedScalarFunction(value = "map_remove_null_values", deterministic = true, calledOnNullInput = true, descriptor = @ScalarFunctionDescriptor(
```

Comment:
> What's the blocker to supporting BIND function?

Time. We have a lot of limitations that need to be addressed later.

Reply:
>Do we need this because we don't support BIND function properly? 

Actually, we do not really need it. SqlInvokedFunction is not a real function and will be substituted with the SQL code. Good catch!